ballsdex/core/admin/resources.py

```diff
@@ -228,7 +228,23 @@ class BallInstanceResource(Model):
     icon = "fas fa-atlas"
     page_pre_title = "ball instances list"
     page_title = "Ball instances"
-    filters = []
+    filters = [
+        filters.Search(
+            name="id",
+            label="Ball Instance ID",
+            placeholder="Search for balls",
+        ),
+        filters.Search(
+            name="ball_id",
+            label="Ball ID",
+            placeholder="Search for balls",
+        ),
+        filters.Search(
+            name="player",
+            label="Player ID",
+            placeholder="Search for player",
```

Comment:
You can query relations directly using `__` in some Tortoise fields :P 

I think it would be better if you could search for player Discord ID directly
```suggestion
            name="player__discord_id",
            label="User ID",
            placeholder="Search for Discord user ID",
```